src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -78,17 +105,24 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 			}
 		}
 
-		let tooltip = 'Server: ' + connectionProfile.serverName + '\r\n' +
-			'Database: ' + (connectionProfile.databaseName ? connectionProfile.databaseName : '<default>') + '\r\n';
+		let tooltip = localize('status.connection.baseTooltip', 'Server: {0}\r\nDatabase: {1}\r\n', connectionProfile.serverName,
```

Comment:
nit: Remove the `\r` - we're crossplatform so we shouldn't be using the OS-specific newline (and `\n` is enough for it to be displayed properly on all platforms)